src/gf.c

```diff
@@ -388,14 +388,15 @@ jl_code_info_t *jl_type_infer(jl_method_instance_t *mi, size_t world, int force)
     }
     JL_CATCH {
         jl_value_t *e = jl_current_exception();
+        jl_printf((JL_STREAM*)STDERR_FILENO, "Internal error during type inference of\n");
```

Comment:
Maybe align with the other `Internal error: ...` reporting we have? https://github.com/JuliaLang/julia/blob/fb8d7f1034e6d83c78a7e1f094d5767268603579/src/codegen.cpp#L9164